haskell-src/Concordium/Types/Queries/Tokens.hs

```diff
@@ -77,14 +79,47 @@ data TokenState = TokenState
       -- | The governance account for the token.
       tsIssuer :: !AccountAddress,
       -- | The number of decimals in the token representation.
-      tsDecimals :: !Word8,
+      tsDecimals :: !Word32,
```

Comment:
Why this change? As far as I know, the upper bound for token decimals is 255?

Reply:
It should be kept as `Word8`. Protobuf doesn't have a `uint8`, so it's encoded there as a `unit32`, but we require it to be within the 0-255 range.